src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -1,10 +1,19 @@
 package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import javax.validation.constraints.NotNull;
 
+import static java.util.stream.Collectors.toMap;
+
 public class ServiceConfigItem {
 
+    private static final Logger log = LoggerFactory.getLogger(ServiceConfigItem.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The constant name 'log' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)